bytes/src/main/scala/io/iohk/ethereum/utils/ByteStringUtils.scala

```diff
@@ -2,11 +2,68 @@ package io.iohk.ethereum.utils
 
 import akka.util.ByteString
 
+import scala.collection.mutable
+import scala.math.Ordering.Implicits._
+
 object ByteStringUtils {
   def hash2string(hash: ByteString): String =
     Hex.toHexString(hash.toArray[Byte])
 
   // unsafe
   def string2hash(hash: String): ByteString =
     ByteString(Hex.decode(hash))
+
+
+  implicit class Padding(val bs: ByteString) extends AnyVal {
```

Comment:
Are these additions related to the Scala 2.13 migration? I don't understand how.

Reply:
`padTo` method now belongs to `SeqOps` and the result is no longer a `ByteString`, but an `IndexedSeq`.
We can't apply padding to a `ByteString` using this method any longer, so we have to offer another way.